### PR TITLE
CP-2292 [E2E][HomePage] Device Not Connecting Path

### PR DESCRIPTION
**Pull request by robertmudi**
JIRA Reference: [CP-2292]

### :memo: Description ️
This works on version 2.2.8-dev.995 or higher

https://nexus.mudita.com/#browse/browse:mudita-center:releases%2Ffeature-branch%2FCP-2583%2F2.2.8-dev.995

Test that automatically checks Contact Support Modal while device is not connected.

### :exclamation: Checklist before merging a pull request
- change went through the QA process

[CP-2292]: https://appnroll.atlassian.net/browse/CP-2292?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]
Many changes requested. Lets meet 

0. Test doesn't work on 2.2.8 - Its expected but not written in Pull Request
1. variable naming probably a lot of fixes
2. there are no assertions for the Contact Support form fields (he just clicks on them and "he saw that the cursor was selected" but he did not make the assertion)
4. It can be done better
 const connectionProblemInstructions = await HomePage.weAreSorryPageInstructions
```
        const connectionProblemInstructions1 = await HomePage.weAreSorryPageInstructions1
        const connectionProblemInstructions2 = await HomePage.weAreSorryPageInstructions2
        const connectionProblemInstructions3 = await HomePage.weAreSorryPageInstructions3
        await expect(connectionProblemInstructions).toHaveText('Disconnect the device from the computer')
        await expect(connectionProblemInstructions1).toHaveText('Restart the device'),
        await expect(connectionProblemInstructions2).toHaveText('Reconnect the device to the computer'),
        await expect(connectionProblemInstructions3).toHaveText("Unlock your Mudita device's screen (if applicable)");
```
5. You use 4 spaces instead of 2 spaces (indentation)
Prettier doesn't work
6. We have an error, not necessarily your fault, but unfortunately it needs to be corrected
```
mudita-center-e2e/src/specs/settings/mc-version-check-for-updates.e2e.ts
Property 'notNowButton' does not exist on type 'HomePage'.
```

7.  github checks
```
***) Home Screen Page Click on My Device doesn't show up
[chrome ***6.***.58***5.228 linux #***-2] **melement click intercepted: Element <p class="sc-bcXHqe gvwApy sc-ftTHYK ilZBiE" color="primary">...</p> is not clickable at point (639, 767). Other element would receive the click: <div class="ReactModal__Overlay ReactModal__Overlay--after-open" style="position: fixed; inset: ***px; background-color: rgba(***, ***, ***, ***.3); z-index: ***2;">...</div>
```
8. The home-page.e2e.ts file should be deleted, it was just an example
9. committed default-news.json

**Comment by tomaszmaleckimudita** [comment]
Blocked by: https://appnroll.atlassian.net/browse/CP-2703

**Review by tomaszmaleckimudita — APPROVED** [review]

**Comment by tomaszmaleckimudita** [comment]
@dkarski  Please research - only Github Actions are failing. Same command on linux works fine 
`npm run e2e:test:linux` 
`./Mudita-Center.AppImage --appimage-extract`
`zvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' nx e2e:test:cicd mudita-center-e2e`

File: apps/mudita-center-e2e/src/specs/overview/home-page-device-not-connecting.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import HomePage from "../../page-objects/home.page"
+import { sleep } from "../../helpers/sleep.helper"
```
Comment (dkarski) [inline apps/mudita-center-e2e/src/specs/overview/home-page-device-not-connecting.e2e.ts:2]: not use so can be delete
```suggestion
```

File: apps/mudita-center-e2e/src/specs/overview/home-page-device-not-connecting.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+describe("Home Screen Page", () => {
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 10000)
+    })
+  })
```
Comment (dkarski) [inline apps/mudita-center-e2e/src/specs/overview/home-page-device-not-connecting.e2e.ts:10]: no needed here
```suggestion
```

**Review by tomaszmaleckimudita — DISMISSED** [review]

File: apps/mudita-center-e2e/src/page-objects/home.page.ts [hunk 3]
```diff
@@ -7,12 +7,71 @@ import Page from "./page"
...
-    return $("h2*=Welcome to Mudita Center")
+    return $("h1*=Welcome to Mudita Center")
   }
 
   get notNowButton() {
     return $("button*=Not now")
   }
-}
 
+  get myDevicesDoesntShowButton() {
```
Comment (lkowalczyk87) [inline apps/mudita-center-e2e/src/page-objects/home.page.ts:17]: The getter name does not correspond to the return value. The name suggests a button, but you are returning a paragraph. 

The same comment applies to other getters. (some of them are missing the type of returned element)

Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/page-objects/home.page.ts:17]: Now it is really the button. 

Others fixed

**Review by tomaszmaleckimudita — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**